views/detail/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = view.extend({
     id: 'detail',
     template: require('./index.html'),
     data: {
-        back: true,
+        back: '/templates',
```

Comment:
This is going to cause an old issue to resurface which is that users will always return to `/templates` regardless of the starting position. As per spec, a user can enter the detail view through discovery or through templates.